frontend/src/css/modules/analytics.scss

```diff
@@ -0,0 +1,38 @@
+li.analytics-list {
```

Comment:
@aayusharora here .analytics-list is not a child of list element then how can you write this inside the nested loop?